modules/users/client/views/verify/verify-account.client.view.html

```diff
@@ -0,0 +1,14 @@
+<section style="margin-top:50px;" class="row text-center" ng-init="vm.validateVerifyToken()">
```

Comment:
Can we remove the `ng-init` attribute from this directive? I think it would be better to initiate `validateVerifyToken()` from the controller.